crates/forge/src/run_tests/package.rs

```diff
@@ -131,9 +136,9 @@ pub async fn run_for_package(
     }: RunForPackageArgs,
     block_number_map: &mut BlockNumberMap,
 ) -> Result<Vec<TestTargetSummary>> {
+    clean_up_forge_test_filter();
```

Comment:
Why do we call this twice?

Reply:
Also, not sure if this should be called here. We execute this function multiple times, asynchronously. I think we should be unsetting the filter at the level we are setting it, before we return an `ExitStatus`.